### DISCUSSION TITLE
put in new smplayer  i cant fint the   how to add smtube to too bar or where the shortcut is

### DISCUSSION THREAD

**Discussion by Cootmaster** [smplayer-dev/smplayer#964] [Q&A]
i put new smplayer in  and i cant  find the smtube in tool bar or short cuts.
i put it on desktop.  where is smtube in tool bar and shortcuts?

**Comment by smplayer-dev** [comment]
In version 24.5, the smtube button has been removed because smtube is obsolete and it will be replaced with something else in the future.

**Comment by Cootmaster** [comment]
ok ty, as for my other thread, why  is chrome /edge not allowing downloads/blocking
also i noticed the icon is a piece of paper  . or was
